Apps/phproofpoint/proofpoint.json

```diff
@@ -845,6 +845,89 @@
                 }
             ],
             "versions": "EQ(*)"
+        },
+        {
+            "action": "decode url",
+            "description": "Decode Proofpoint rewrote URL(s)",
```

Comment:
rewrote --> rewritten

Reply:
Might have to delete this PR and redo it. It is not accepting my changes. I tried to rebase to this PR then change the files and still nothing.

Reply:
Okay no problem. Let me know if you want me to close this PR.

Reply:
@ricardo-phantom Yeah please close and I will redo the PR.

Reply:
Closed